types/styled-system/index.d.ts

```diff
@@ -709,20 +808,51 @@ export interface GridAreaProps {
 export const gridArea: styleFn;
 
 /**
- * Borders
+ * A convenience style group containing props related to grid.
+ *
+ * - String values are passed as raw CSS values.
+ * - Array values are converted into responsive values.
  */
+export interface GridProps
+    extends GridGapProps,
+        GridColumnGapProps,
+        GridRowGapProps,
+        GridColumnProps,
+        GridRowProps,
+        GridAutoFlowProps,
+        GridAutoColumnsProps,
+        GridAutoRowsProps,
+        GridTemplateColumnsProps,
+        GridTemplateRowsProps,
+        GridTemplatesAreasProps,
```

Comment:
I think this has been an ongoing typo that just keeps popping up. Shouldn't this be:

```suggestion
        GridTemplateAreasProps,
```

Reply:
Also on line 786

Reply:
Don't forget to refactor the other misspelled references as well 👍 thanks!

Reply:
Thanks for the catch, @rathpc. Refactored other references and added a test.